database/factories/PostFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Post>
+ */
+class PostFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        $w = fake()->word();
+        return [
+            'title' => $w,                      /* default one word */
+            'content' => fake()->sentence(2),   /* default 160 characters */
+            'description' => $w.'.JPG',
+            'created_at' => fake()->dateTimeBetween(),
```

Comment:
dateTimeBetween what? 
Default arguments values is $startDate = '-30 years', $endDate = 'now'
I don't think you need such values